2108.02159/2108.02159.yaml

```diff
@@ -92,71 +98,75 @@
       score:
         Haining: 2
         Will:
-        Yasaman:
+        Yasaman: 2
+        # This includes the potential too, but there’s no alternative symbol in the excerpt
     parabolic|Dirac|cos:
       LLM: parabolic
       human:
       score:
         Haining: 2
         Will:
-        Yasaman:
+        Yasaman: 2
     electrons|holes:
       LLM: electrons
       human: holes
       score:
         Haining: 0
         Will:
-        Yasaman:
+        Yasaman: 1
+        # Partial credit, since excerpt also says “T is for electrons”
```

Comment:
I agree with the existence of this sentence, but a minus sign before Eq. (3) makes me think it's hole-like dispersion. 

But it's true that these contradictory parts will confuse LLM